include/dt-bindings/pinctrl/stm32-pinctrl.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _STM32_PINCTRL_H_
+#define _STM32_PINCTRL_H_
+
+#include <dt-bindings/pinctrl/stm32-pinctrl-common.h>
+/**
+ * @brief PIN configuration bitfield
+ *
+ * Pin configuration is coded with the following
+ * fields
+ *    Alternate Function  [ 0 : 7 ]
+ *    GPIO Mode           [ 8 : 9 ]
+ *    GPIO Output type    [ 10 ]
+ *    GPIO Speed          [ 11 : 12 ]
+ *    GPIO PUPD config    [ 13 : 14 ]
+ *
+ * Applicable to STM32F3, STM32F4, STM32L4 series
+ */
+
+#define STM32_AFR_MASK			0xF
```

Comment:
If it's really 8 bits, isn't the mask 0xFF, not 0xF? Though the rest of the file makes it look like it's really just 4 bits, and the comment is wrong, should be:

```
 *    Alternate Function  [ 0 : 3 ]
 *    GPIO Mode           [ 4 : 5 ]
 *    GPIO Output type    [ 6 ]
 *    GPIO Speed          [ 7 : 8 ]
 *    GPIO PUPD config    [ 9 : 10 ]
```